jans-pycloudlib/docs/api/persistence/cloudsql_connector.md

```
@@ -0,0 +1,401 @@
+
+2. **Check Logs**: Review Cloud Run logs for connection errors or IAM permission issues.
+
+3. **Verify IAM Bindings**:
+
+```bash
+gcloud projects get-iam-policy PROJECT_ID \
+  --flatten="bindings[].members" \
+  --filter="bindings.role:roles/cloudsql.client"
+```
+
+4. **Verify VPC Connector Status**:
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)